documentation/gapc on windows.txt

```diff
@@ -0,0 +1,23 @@
+How to compile gapc on Windows 10:
+
+1) Install Linux Sub-System
+	1.1) go Start -> Settings -> For Developers : Click Developer-Mode
+	1.2) Open Windows Power Shell then type: Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
+	1.3) restart computer
+	1.4) Download Ubuntu app from Windows-Strore
+	1.5) Initialize installation by setting Username and Password
+2) Prepare compiling and installation of gapc by installing requiered sources: 
+	2.1) Open Ubuntu app

```

Comment:
you here might want to point to the .travis file in this repo. That should instruct on which packages to install